### ISSUE TITLE
fix: Set asynchronously properties from `useMeta` does not react for the first load

### ISSUE THREAD

**Issue by KevinNTH** [nuxt-community/composition-api#78]
**🐛 The bug**
I try to set the `title` asynchronously:

```javascript
    const { title } = useMeta({ title: 'Page' })
    setTimeout(() => {
      title.value = 'My page'
    }, 1000)
```

When navigating to the page for the first time, the page title is "Page" and does not react to be "My page" after 1 second as it should be. Navigating again to the page (without browser hard reload), the page title is cached and set to be "My page" if [`keep-alive`](https://nuxtjs.org/api/pages-fetch/#caching) is configured, otherwise the page title would remain the same.

When the meta properties are cached, it's unconvenient for dynamic routes such as `/blog/_slug.vue` where content are fetched asynchronously. In this case, navigating to `/blog/article-2` with title page "Article 2" after visiting `/blog/article-1` display cached page title "Article 1"

**🛠️ To reproduce**

1. Set [`keep-alive`](https://nuxtjs.org/api/pages-fetch/#caching)
2. Hard reload browser on `/blog/`
3. Navigate to `/blog/article-1`
4. Page title is not set
5. Go back to `/blog/`
6. Navigate to `/blog/article-1` again
7. Page title is set to "Article 1"
8. Navigate to `/blog/article-2`
9. Page title is not set (first time visiting this page)
10. Go back to `/blog/`
11. Navigate to `/blog/article-2`
12. Page title is set to "Article 1" because of [`keep-alive`](https://nuxtjs.org/api/pages-fetch/#caching)
13. Implement [`activated hook`](https://nuxtjs.org/api/pages-fetch/#using-code-activated-code-hook) to set `this.title` from `useMeta()` => `this.title = this.post?.title ?? 'Default title'` to fix step 12. 
14. Unfortunately, step 13 does not fix steps 4 and 9.

**🌈 Expected behaviour**
I expect `ref` properties from `useMeta` to be reactive.
Needing to use [`activated hook`](https://nuxtjs.org/api/pages-fetch/#using-code-activated-code-hook) along with [`keep-alive`](https://nuxtjs.org/api/pages-fetch/#caching) to update the meta data is a _gotcha_ so I don't know if it's intended. If so, add it into the documentation?

**ℹ️ Additional context**
I am _still using_ `apollo-client` with `@vue/apollo-composable` instead of `useFetch()` to fetch the post data.

**Comment by danielroe** [comment]
More information [here](https://github.com/nuxt-community/composition-api/pull/65#issuecomment-631527341). This needs to be fixed - and thanks for reporting 😃

**Comment by danielroe** [comment]
This should be fixed in v0.7.3 but please do confirm - and re-open if this isn't the case!

**Comment by KevinNTH** [comment]
Fixed! Thanks a lot! 🙏
